composer.json

```diff
@@ -40,7 +40,8 @@
         "spoon/library": "^3.0",
         "league/flysystem-aws-s3-v3": "^1.0.13",
         "league/flysystem-cached-adapter": "^1.0.6",
-        "pimple/pimple": "^3.2"
+        "pimple/pimple": "^3.2",
+        "jeroendesloovere/geolocation-php-api": "^2.1"
```

Comment:
why did you create your own package instead of using an existing/proven one?

Reply:
Can you point me in the direction of such a library?

Reply:
https://packagist.org/